spec/models/article_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  feature 'admin creates an article' do
```

Comment:
`feature` should only be used for feature specs using capybara, use `describe` instead